migration_scripts/add_drs_step_and_roi.py

```diff
@@ -0,0 +1,29 @@
+from erna.automatic_processing.database import database
+from erna.utils import load_config
+import click
+
+from playhouse.migrate import MySQLMigrator, migrate
+from peewee import IntegerField, CharField
+
+
+@click.command()
+@click.option('--config', '-c', help='Path to the yaml config file')
+def main(config):
```

Comment:
I don't think this is a goodway to track schema changes over time. Maybe we can use something like this? https://github.com/klen/peewee_migrate